internal/removals/remove.go

```diff
@@ -126,15 +126,18 @@ func GetRemainingMap(logger logr.Logger, adminClient fdbadminclient.AdminClient,
 			continue
 		}
 
-		if len(processGroup.Addresses) == 0 {
-			logger.Info("Getting remaining removals to check for exclusion", "processGroupID", processGroup.ProcessGroupID, "reason", "missing address")
+		// If we use localities for exclusions we don't have to care about the addresses.
+		if cluster.UseLocalitiesForExclusion() {
```

Comment:
The use localities for exclusion must have an additional check for the released version.

Reply:
We have to add the IP addresses for log processes, in case they might be missing in the status.